pyscitt/pyscitt/cli/submit_signed_claims.py

```diff
@@ -48,17 +60,26 @@ def submit_signed_claimset(
 
 def cli(fn):
     parser = fn(
-        description="Submit signed claimset to a SCITT CCF Ledger and retrieve receipt"
+        description="Submit signed claimset (COSE) to a SCITT CCF Ledger and retrieve receipt"
     )
     add_client_arguments(parser, with_auth_token=True)
-    parser.add_argument("path", type=Path, help="Path to signed claimset file")
+    parser.add_argument("path", type=Path, help="Path to signed claimset file (COSE)")
     group = parser.add_mutually_exclusive_group()
     group.add_argument("--receipt", type=Path, help="Output path to receipt file")
     group.add_argument(
         "--skip-confirmation",
         action="store_true",
         help="Don't wait for confirmation or a receipt",
     )
+    parser.add_argument(
+        "--receipt-type",
+        choices=["embedded", "raw"],
```

Comment:
You could use the `ReceiptType` enum values directly here for simplicity:

```suggestion
        choices=[e.value for e in ReceiptType],
```

Reply:
done